mantidproject/mantid
# Adding a help button to the sans V2 GUI

A help button has been added to the lower left hand corner of the new sans GUI. The Icons on the left have also been reordered so that settings is second and new icons have been given to the other tabs.

**To test:**

Open the Sans Gui, confirm that settings is now second after runs in the list on the left.

Click on the help button on the bottom left and confirm that documentation is opened.



Fixes #21931  

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
